### PR TITLE
Bug 1164796 - [Video] Migrate to tagged template strings

### PR DESCRIPTION
**Pull request by KevinGrandon**
https://bugzilla.mozilla.org/show_bug.cgi?id=1164796

### CODE REVIEW

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=83b9dc22f8d0a230897d42ca5fe948c127b13054

File: apps/video/js/thumbnail_item.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-/* global MediaUtils,VideoUtils */
+/* global MediaUtils,Tagged,VideoUtils */
 /**
  * ThumbnailItem is view object for a single video data. It renders video with
- * the ThumbnailItem.Template. Before use it, The template should be
+ * ThumbnailItem.view. Before use it, The template should be
```
Comment (russnicoletti) [inline apps/video/js/thumbnail_item.js]: Nit: it seems this part of the comment no longer makes sense "Before use it, The template should be initialized."

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0cb44c947ccbe28ffb0ea4d52c250f6c1d8937ae

**Comment by lightsofapollo** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7cb2795fa625ab16aaf4d113660e225d8038896e

**Pull request merged**